style.css

```diff
@@ -87,3 +88,278 @@ body {
   justify-content: flex-start;
   padding-left: 5px;
 }
+
+.page1 {
+  background-color: var(--v3);
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-top: 0%;
+  justify-content: center;
+  background-repeat: no-repeat;
+}
+
+.tituloMR {
+  font-style: normal;
+  font-size: 40px;
+  color: var(--v2);
+  font-family: 'Crete Round', serif;
+  padding-left: 15px;
+  margin-bottom: 0%;
+  padding-top: 802px;
+}
+
+.vector {
+  border: 1px solid #212121;
+  color: var(--v2);
+  left: 24px;
+  right: 24px;
+}
+
+.yoga {
+  left: 24px;
+  padding-top: 104px;
+  padding-left: 15px;
+  padding-right: 30px;
+}
+
+.multi {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+  padding-left: 15px;
+  padding-right: 30px;
+}
+
+.parrafo1 {
+  font-family: 'Inter', sans-serif;
+  padding-left: 15px;
+  padding-right: 30 px;
+  font-size: 16px;
+  color: #091e42;
+}
+
+.list_programs {
+  display: flex;
+  align-content: space-between;
+  flex-direction: row;
+  padding: 10px 12px;
+}
+
+.boton1 {
+  height: 48px;
+  background: #ff6b00;
+  margin-left: 15px;
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+.projects {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-image: url(./images/fondo.svg);
+  background-repeat: no-repeat;
+  width: 327px;
+}
+
+.tituloPAPD {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  padding-left: 15px;
+}
+
+.parrafo2 {
+  font-family: 'Inter', sans-serif;
+  padding-left: 15px;
+  padding-right: 30 px;
+  font-size: 16px;
+  color: #fff;
+}
+
+.list_programs2 {
+  display: flex;
+  align-content: space-between;
+  flex-direction: row;
+  padding: 10px 12px;
+}
+
+.boton2 {
+  width: 327px;
+  height: 48px;
+  background: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+.page3 {
+  background-color: var(--v2);
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-top: 0%;
+  justify-content: center;
+  background-image: url(./images/about.svg);
+  background-repeat: no-repeat;
+  background-position-y: bottom;
+  background-position-x: right;
```

Comment:
- [ ] Great on implementing images in the background using CSS property; it's good practice, but make sure the image position sticks with the template you choose.
- [ ] Make sure all your buttons are similar to the template design and all buttons have the button styling: Active, Hover, and Pressed

| Figma template design  | Your design|
| ------------- | ------------- |
|![Screenshot from 2022-10-29 09-53-37](https://user-images.githubusercontent.com/65129228/198818365-440dcd34-ea32-4caa-9d73-905ffe034877.png)|![Screenshot from 2022-10-29 09-52-56](https://user-images.githubusercontent.com/65129228/198818390-4f44a1ca-0a2f-4619-9663-bcd5288dc3f7.png)|